flytekit/common/launch_plan.py

```diff
@@ -225,12 +206,44 @@ def execute_with_literals(self, project, domain, literal_inputs, name=None, noti
 
     @_exception_scopes.system_entry_point
     def __call__(self, *args, **input_map):
-        raise _user_exceptions.FlyteAssertion(
-            "TODO: Implement adding of remote launch plans to workflows. Current workaround is to add remote "
-            "workflows directly."
+        """
+        :param list[T] args: Do not specify.  Kwargs only are supported for this function.
+        :param dict[Text,T] input_map: Map of inputs.  Can be statically defined or OutputReference links.
+        :rtype: flytekit.common.nodes.SdkNode
+        """
+        if len(args) > 0:
+            raise _user_exceptions.FlyteAssertion(
+                "When adding a launchplan as a node in a workflow, all inputs must be specified with kwargs only.  We "
+                "detected {} positional args.".format(self, len(args))
+            )
+
+        # Take the default values from the launch plan
+        default_inputs = {
+            k: v.sdk_default
+            for k, v in _six.iteritems(self.default_inputs.parameters) if not v.required
+        }
+        default_inputs.update(input_map)
+
+        bindings, upstream_nodes = self.interface.create_bindings_for_inputs(default_inputs)
+
+        # TODO: Remove DEADBEEF
```

Comment:
Yes plz. I would take a look at the `rename_and_return` method on the SdkNode I believe, this is where we rewrite the id, it should also set the name in metadata...

Reply:
Yup, i'll just make it empty to start.